src/BlazorUI/Bit.BlazorUI/Components/RadioButtonGroup/BitRadioButtonGroup.razor.cs

```diff
@@ -134,6 +154,38 @@ private void SelectOptionByKey(string? value)
 
     private string GetAriaLabelledBy() => Label.HasValue() || LabelFragment is not null ? LabelId : AriaLabelledBy;
 
+    private void HandleOnValueChanging(object? sender, ValueChangingEventArgs<string?> args)
+    {
+        if (firstRender) return;
```

Comment:
what is this???

Reply:
The first time the Event is called because the value of the `AllOptions` is not yet initialized, the execution of this method should be skipped and this part should be managed by the OnAfterRenderAsync method.

Reply:
you can just check the AllOptions then, no need to introduce another field for this class

Reply:
We do this by checking that the item variable `AllOptions` has an item, when the component does not have an item and the Value has a value, then the Value will not be reset.

Reply:
so remove that redundant variable